src/main/java/com/dev/cinema/service/mapper/MovieSessionMapper.java

```diff
@@ -0,0 +1,34 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.dto.MovieSessionRequestDto;
+import com.dev.cinema.model.dto.MovieSessionResponseDto;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.MovieService;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionMapper {
+    private MovieService movieService;
+    private CinemaHallService cinemaHallService;
```

Comment:
Does it work? I don't see these autowiring

Reply:
@Max-Pochepets constructor-way autowiring implemented below